glass-easel/src/tmpl/proc_gen_wrapper.ts

```diff
@@ -1002,22 +1004,27 @@ export class ProcGenWrapper {
     mutated: boolean,
     capture: boolean,
     isDynamic: boolean,
-    _generalLvaluePath?: DataPath | null,
+    generalLvaluePath?: DataPath | null,
   ) {
-    const handler = typeof v === 'function' ? this.eventListenerFilter(v) : dataValueToString(v)
-    const listener = (ev: ShadowedEvent<unknown>) => {
-      const host = elem.ownerShadowRoot!.getHostNode()
-      let ret: boolean | undefined
-      const methodCaller = host.getMethodCaller() as { [key: string]: unknown }
-      const f = typeof handler === 'function' ? handler : Component.getMethod(host, handler)
-      if (typeof f === 'function') {
-        const filteredEv = this.eventObjectFilter(ev)
-        ret = (f as (ev: ShadowedEvent<unknown>) => boolean | undefined).call(
-          methodCaller,
-          filteredEv,
-        )
+    const handler = typeof v === 'function' ? v : dataValueToString(v)
+    const host = elem.ownerShadowRoot!.getHostNode()
```

Comment:
These logics are move forwarded - may have had impact on tree building process.

Reply:
moved into listener and made a lazy initialization to avoid running filter more than once